src/Integration.Vsix/SonarLintDaemon/SonarLintDaemon.cs

```diff
@@ -131,23 +130,24 @@ public void Start()
             process.OutputDataReceived += (sender, args) =>
             {
                 string data = args.Data;
-                if (data != null)
+                if (data != null && data.Contains("Server started"))
                 {
-                    if (data.Contains("Server started"))
+                    if (IsVerbose())
                     {
-                        try
+                        WritelnToPane(data);
+                    }
+
+                    bool succeeded = false;
+                    SafeOperation(() =>
                         {
                             CreateChannelAndStreamLogs();
                             Ready?.Invoke(this, EventArgs.Empty);
-                        }
-                        catch (Exception ex) when (!ErrorHandler.IsCriticalException(ex))
-                        {
-                            logger.WriteLine($"Daemon error: {ex.ToString()}");
-                        }
-                    }
-                    if (IsVerbose())
+                            succeeded = true;
+                        });
+
+                    if (!succeeded)
                     {
-                        WritelnToPane(data);
+                        SafeOperation(() => this.Stop());
```

Comment:
If we don't stop the daemon, our code will continue to run inside VS whenever a user opens a file, leading to further errors. So, we might as well explicitly stop the daemon.